server/netty-app/build.gradle

```diff
@@ -76,12 +76,6 @@ if (hasProperty('quiet')) {
 tasks.withType(JavaExec).configureEach {
     // This appends to the existing jvm args, so that java-open-nio still takes effect
     jvmArgs extraJvmArgs
-
-    jvmArgs '-XX:+UseG1GC',
-            '-XX:MaxGCPauseMillis=100',
-            '-XX:+UseStringDeduplication',
-            // For development in this repository via Gradle, use the top-level data directory
-            "-Dstorage.path=${rootDir}/data"
```

Comment:
We don't need/want this anymore?

Reply:
The defaults should be good enough IMO.

The equivalent was removed from jetty-app, and was accidentally left behind here. storage.path defaults to `<deephaven.dataDir>/storage`, and for local development (prepareCompose, docker compose up) that equates to `${rootDir}/data/storage` given:

```Dockerfile
    volumes:
      - ./data:/data
```

For native local development (`./gradlew ... :run`), the default will use OS conventional paths.

Reply:
Might changing this break persisted notebooks for users? Do we need to put out a notice of the change?

Reply:
This image is used for local development only.